frontend/app/routes/$lang/_public/status/child.tsx

```diff
@@ -294,6 +296,9 @@ export default function StatusCheckerChild() {
           <ButtonLink id="cancel-button" variant="primary" type="button" routeId="$lang/_public/status/index" params={params} className="mt-12" endIcon={faChevronRight}>
             {t('status:child.check-another')}
           </ButtonLink>
+          <InlineLink routeId="$lang/_public/status/index" params={params} className="mt-6 block font-lato font-semibold">
```

Comment:
The link's URL should be `https://www.canada.ca/en/services/benefits/dental/dental-care-plan/apply.html#status` in English or `https://www.canada.ca/fr/services/prestations/dentaire/regime-soins-dentaires/demande.html#etat` in French